publish.js

```diff
@@ -0,0 +1,14 @@
+var ghpages = require('gh-pages');
+
+ghpages.publish('public', {
+  user: {
+    name: 'OpenFisca-Bot',
+  },
+  branch: 'gh-pages',
+  message: 'Auto-commit from master branch'
+}, function(err) {
+  if(err){
+  console.error(err)
+  process.exit(1)
```

Comment:
Improper indentation.

Since this is the second time on this PR, I would recommend adding a linter to catch these errors early 🙂